File: src/nl/rubensten/texifyidea/action/insert/InsertParagraphAction.java
package nl.rubensten.texifyidea.action.insert;

import nl.rubensten.texifyidea.TexifyIcons;
import nl.rubensten.texifyidea.action.InsertEditorAction;

/**
 * @author Ruben Schellekens
 */
public class InsertParagraphAction extends InsertEditorAction {

    public InsertParagraphAction() {
        super("Paragraph", TexifyIcons.DOT_PARAGRAPH, "\\paragraph{", "}");
    }
}

File: src/nl/hannahsten/texifyidea/action/insert/InsertParagraphAction.kt
package nl.hannahsten.texifyidea.action.insert;

import nl.hannahsten.texifyidea.TexifyIcons;
import nl.hannahsten.texifyidea.action.InsertEditorAction;

/**
 * @author Hannah Schellekens
 */
public class InsertParagraphAction extends InsertEditorAction {

    public InsertParagraphAction() {
        super("Paragraph", TexifyIcons.DOT_PARAGRAPH, "\\paragraph{", "}");
    }
}
